src/lib/react/React.hx

```diff
@@ -87,3 +93,9 @@ extern interface ReactChildren
 }
 
 typedef CreateElementType = haxe.extern.EitherType<haxe.extern.EitherType<String, haxe.Constraints.Function>, Class<ReactComponent>>;
+
+typedef ReactContext<T> = {
+	var displayName:String;
+	var Provider:Dynamic->ReactElement;
```

Comment:
More like `{value:T}->ReactElement`, though it doesn't really matter for jsx on haxe-react.